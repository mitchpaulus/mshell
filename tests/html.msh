"<html name=\"html-name\">
<head>
    <title>My HTML Document</title>
</head>
<body>
    <p>This is a simple HTML document.</p>
</body>
</html>" parseHtml html!

# @html str wl
@html "children" get? :0: # HTML tag
"children" get? :0: head! @head "tag" get? wl # head

@head "children" get? :0: title!
@title "tag" get? wl
@title "text" get? wl
