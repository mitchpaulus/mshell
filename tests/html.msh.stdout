head
title
My HTML Document
Testing descendents..
title
head
p
body
html

